android/res/layout/view_vpn_status_detail.xml

```diff
@@ -48,49 +48,130 @@
             android:gravity="center_horizontal"
-            android:text="@string/unprotected_connection" />
+            android:text="@string/connection_header_title" />
 
-        <!-- Get VPN Button -->
+        <TextView
+            android:id="@+id/view_vpn_status_header_un_protected"
+            style="@style/VPNTitle"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:gravity="center_horizontal" />
+
+        <!-- VPN description text -->
+        <TextView
+            android:id="@+id/VPNText"
+            style="@style/VPNText"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginTop="18dp"
+            android:text="@string/unprotected_connections_visibility_bullet_html" />
+
+        <!-- Unprotected connection button -->
         <Button
             android:id="@+id/view_vpn_status_get_vpn_button"
             style="@style/GetVPNButton"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center"
-            android:layout_marginBottom="15dp"
-            android:layout_marginTop="15dp"
-            android:paddingLeft="15dp"
-            android:paddingRight="15dp"
-            android:text="@string/protect_my_privacy" />
+            android:layout_marginTop="30dp" />
 
-        <!-- privacy as a human right paragraph -->
         <TextView
-            style="@style/VPNSubTitle"
-            android:layout_width="fill_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="15dp"
-            android:padding="0dp"
-            android:text="@string/your_privacy_is_a_fundamental_right_html" />
+            android:id="@+id/view_vpn_money_back"
+            style="@style/VPNSmallText"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:gravity="center_horizontal" />
 
-        <!-- explanation about what is open -->
         <TextView
-            android:id="@+id/VPNText"
-            style="@style/VPNText"
+            android:id="@+id/view_vpn_price"
+            style="@style/VPNSmallText"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:layout_marginTop="15dp"
-            android:text="@string/unprotected_connections_visibility_bullet" />
+            android:gravity="center_horizontal" />
+
+        <!-- Protected connection button -->
+        <Button
+            android:id="@+id/view_vpn_status_learn_more_button"
+            style="@style/GetVPNButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:layout_marginTop="30dp" />
+
+
+        <ImageView
+            android:id="@+id/imageView"
```

Comment:
does this need an id? if so, please make the name self explanatory, perhaps:
`rename to `view_vpn_status_divider_image_view`. If you are not using this view in your java code, I believe you can just get rid of the id.